src/query/index.ts

```diff
@@ -98,14 +105,17 @@ const filterToApiQueryFilter = (_filters: QueryStoreFilter[], timezone = 'UTC')
 export class QueryHelper {
     private static timezone: ComputedRef<string> | undefined;
 
+    private static referenceStore: ComputedRef<Object> | undefined;
```

Comment:
비교적 명확한 타입선언이 있으면 더 좋을 것 같습니다!
```suggestion
  type ReferenceMap = <string, { label: string; name: string }>
  type ReferenceStore = Record<string, ReferenceMap>
    private static referenceStore: ComputedRef<ReferenceStore> | undefined;
```